omod/src/main/resources/config.xml

```diff
@@ -11,10 +11,10 @@
 		See <a href="http://openmrs.org/wiki/Idgen_Module">http://openmrs.org/wiki/Idgen_Module</a> for more information
 	</description>
 	<activator>@MODULE_PACKAGE@.IdgenModuleActivator</activator>
-	<require_version>1.9.2</require_version>
 	<require_modules>
 		<require_module>org.openmrs.module.webservices.rest</require_module>
 	</require_modules>
+	<require_version>1.11.3, 1.10.2 - 1.10.*, 1.9.9 - 1.9.*</require_version>
```

Comment:
@dkayiwa @wluyima @rkorytkowski  - just curious as to your thoughts on this.  Are we recommending the above and is it necessary?  Would just having 1.9.9,1.10.2,1.11.3 be sufficient, without the wildcard ranges, or are they needed?